apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -17,13 +144,15 @@
   }
 }
 
-.reactions-container {
-  background-color: #f2f2f2;
-  padding: 20px;
+.webcam-container {
+  margin-bottom: 10px;
+  background-color: #ffffff;
```

Comment:
Color `#ffffff` should be written as `#fff`

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexlength)